src/components/Stories/README.md

```diff
@@ -0,0 +1,47 @@
+## Stories
+
+Component for displaying stories.
+
+### PropTypes
+
+| Property        | Type       | Required | Default | Description                              |
+| :-------------- | :--------- | :------- | :------ | :--------------------------------------- |
+| open            | `Boolean`  | ✓        |         | Visibility flag                          |
+| stories         | `Story[]`  | ✓        |         | List of stories to display               |
```

Comment:
Isn't it would be more flexible to allow "stories" as children?

```jsx
<Stories>
  <Story title="New navigation" description="Description">
    <Video/>
  </Story>
</Stories>
```

Reply:
It seems that now such flexibility is not needed and it will complicate the code. We can add later if needed.